apps/chat/src/store/settings/settings.epic.ts

```diff
@@ -34,6 +35,29 @@ import { ShareActions } from '../share/share.reducers';
 import { UIActions } from '../ui/ui.reducers';
 import { SettingsActions, SettingsSelectors } from './settings.reducers';
 
+const getInitActions = (page?: PageType) => {
+  switch (page) {
+    case PageType.Marketplace:
```

Comment:
What if i have fromConversation param initially? Will it be applyed to selected conversation?

Reply:
It will be applyed, can be checked on review env, works fine